packages/react-core/src/components/InputGroup/examples/InputGroupWithDropdown.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Button, InputGroup, TextInput, Dropdown, DropdownToggle, DropdownItem } from '@patternfly/react-core';
+
+export const InputGroupWithDropdown: React.FunctionComponent = () => {
+  const [isOpen, setIsOpen] = React.useState<boolean>(false);
+  const [selected, setSelected] = React.useState<string>('');
+
+  const onToggle = (isOpen: boolean) => {
+    setIsOpen(isOpen);
+  };
+
+  const onSelect = event => {
```

Comment:
If you keep this function after addressing eric's comment, could you add a type to the params of functions you pass to components? This helps us sanity check that our component props are typed correctly and also helps communicate types easier to people using the component.